tests/test_gridsearch.py

```diff
@@ -114,13 +113,12 @@ def test_grid_search_no_score(self):
         # in wrong results. This only happens with threads, not processes/sync.
         # For now, we'll fit using the sync scheduler.
         grid_search = TuneGridSearchCV(
-            clf, {"C": Cs}, scoring="accuracy", scheduler=MedianStoppingRule())
```

Comment:
Why didn't we replace scheduler with early_stopping here?

Reply:
We could, but in spirit of performing the original tests I decided to remove it.